recipes/heapdict/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "1.0.0" %}
+{% set name = "heapdict" %}
+
+package:
+  name: {{ name|lower }}
+  version: "{{ version }}"
+
+source:
+  fn: heapdict-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz 
+  md5: 47320e136156593c84072d09e770800f
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - heapdict
+
+about:
+    home: http://stutzbachenterprises.com/
```

Comment:
BTW here is the problem. The indentation is wrong :wink: 

To want me to finish this recipe for you?